FAQ.md

```diff
@@ -58,24 +58,22 @@ insufficient, a new publisher with a larger `max_slice_len` can be created.
 
 <!-- markdownlint-disable -->
 
-> [!IMPORTANT]
-> Be aware that the history of the old publisher is lost when it is
+> [!IMPORTANT] Be aware that the history of the old publisher is lost when it is
```

Comment:
Oh no, does the script have some autocorrection?
This breaks the github [alerts syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts). `> [!IMPORTANT]` must be on a separate line than the content

Reply:
@zmostafa can you please revert those changes. It breaks the alert syntax. Please also have a look at the other open point.

Reply:
@elBoberido hmmm, I think i did, are you sure you are looking at the latest commit?

Reply:
@zmostafa I hope to look at the latest commit :sweat_smile:

I did a Ctrl+F5, which should reload everything and dismiss caches but I still see `> [!IMPORTANT]` on the same line as `Be aware that ...`

Reply:
Note I believe this happens because the auto-corrector does not recognise GitHub-style admonitions, so you need to manually ensure these are on their own line otherwise the auto corrector will try and do line length correction with it on the same line.

Not ideal, I know, but the idea is for the corrector to fix around 90% of the issues which should save more time than manually fixing these small edge cases (e.g. manually fixing all line lengths).

Reply:
Can we skip line breaks in the auto-corrector? It's just a styling issue, not a rendering issue and it seems to break the github admonitions.

Reply:
I believe this is the line length corrector causing this. If disabled, then users would need to ensure line length limits were maintained, either manually or with an IDE. I deduced that this would be a larger annoyance for contributor than fixing these more limited edge cases. 

Of course, we could check if the auto corrector could be updated to recognise the GitHub admonitions (these were not used in our documentation at the time of preparing the script).

Reply:
Well, I would argue that one is not only an annoyance but breaks the rendering of the readme if the reviewer does not pay attention and the other one is a choice by ourselves and does not break the rendering if the line is too long. I would rather deactivate the line length detection than having to keep in mind to check for broken admonitions in the PR.